frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -139,15 +139,31 @@ object Interpreter {
     state.mergeStatus(ExitStatus.Ok)
   }
 
+  private def compileAnd(
+      state: State,
+      project: Project,
+      reporterConfig: ReporterConfig,
+      excludeRoot: Boolean
+  )(next: State => Task[State]): Task[State] = {
+    Tasks.compile(state, project, reporterConfig, excludeRoot).flatMap { compiled =>
+      if (compiled.status != ExitStatus.CompilationError) next(compiled)
```

Comment:
Shouldn't we only continue if the status is `Ok`?

Reply:
I'm a little bit afraid of doing that now because of https://github.com/scalacenter/bloop/issues/195 and other errors we've had in this area (which is probably the less tested overall in bloop), so I'd prefer to keep it this way for now.